internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -395,8 +407,138 @@ func mutatingToWebhookDescription(webhook admissionregv1.MutatingWebhook, depNam
 	return description
 }
 
+// The default AdmissionReviewVersions set in a CSV for conversion webhooks
+// if ConversionReviewVersions is not specified in CRDs.
+var defaultConversionReviewVersions = []string{"v1beta1", "v1"}
+
+// conversionToWebhookDescription builds a webhookDescription for conversion webhook with the
+// WebhookClientConfig specified in either v1 or v1beta1 CRDs.
+func conversionToWebhookDescription(crdNames []string, v1WebhookClientConfig *apiextv1.WebhookConversion, v1beta1WebhookClientConfig *apiextv1beta1.CustomResourceConversion, depName string, ws *corev1.Service) operatorsv1alpha1.WebhookDescription {
+
+	conversionReviewVersions, whPath, serviceRefName, serviceRefPort := getConversionWebhookDetails(v1WebhookClientConfig, v1beta1WebhookClientConfig)
+
+	description := operatorsv1alpha1.WebhookDescription{
+		Type:                    operatorsv1alpha1.ConversionWebhook,
+		ConversionCRDs:          crdNames,
+		AdmissionReviewVersions: conversionReviewVersions,
+		WebhookPath:             &whPath,
+		DeploymentName:          depName,
+		SideEffects: func() *admissionregv1.SideEffectClass {
+			seNone := admissionregv1.SideEffectClassNone
+			return &seNone
+		}(),
+	}
+
+	if len(description.AdmissionReviewVersions) == 0 {
+		description.AdmissionReviewVersions = defaultConversionReviewVersions
+	}
```

Comment:
admissionReviewVersions != conversionReviewVersions, but it looks like OLM conflates the two terms. @awgreene this field should probably be renamed to `reviewVersions` with a comment about it holding either admission or conversion review versions.

Reply:
Also if these aren't set the generator should not default them. OLM should surface errors when these aren't set
```suggestion
```

Reply:
I agree that the `description.AdmissionReviewVersions` field is misleading and support the proposal to rename the field to `reviewVersions`, but I do not think it will be addressed until OLM releases a new version of the CSV CRD as this is not a backwards compatible change.

Reply:
Would it be possible to deprecate the current field and add the new one now?

Reply:
That seems fine, can you create a GitHub issue against OLM for tracking purposes.

Reply:
Is there a way to mark a field as deprecated using kubebuilder / sdk? [Existing documentation](https://book.kubebuilder.io/reference/markers/crd-validation.html) does not have a deprecated marker.

Reply:
https://github.com/operator-framework/api/pull/113

Reply:
Ignore this for now. Once https://github.com/operator-framework/api/pull/113 is merged and bumped in OLM in a new release, this logic can be updated.